public/javascripts/main.js

```diff
@@ -0,0 +1,19 @@
+$(function () {
+    var _cache = {
+        date: $(".js-birthday__wrapper .date"),
+        month: $(".js-birthday__wrapper .month"),
+    };
+    _cache.month.on('change', function () {
+        if (["2"].indexOf(_cache.month.children("option:selected").val()) !== -1) {
+            _cache.date.children("option.gt28").hide();
```

Comment:
Instead of ``show`` and ``hide``, use ``removeClass('d-none')`` and ``addClass('d-none')``. If you are using bootstrap 3 then use ``removeClass('hidden')`` and ``addClass('hidden')``. ``show`` and ``hide`` puts inline CSS which we should avoid as much as possible.